data/remnant missions.txt

```diff
@@ -493,5 +558,748 @@ mission "Remnant: Technology Available"
+					defer
+
+	on accept
+		log "Volunteered to show Dawn and some of her fellow researchers the world where humans and Quarg co-exist."
+	
+	npc "scan outfits"
+		government Quarg
+		personality staying uninterested
+		fleet Quarg
+		conversation
+			`	Dawn and her compatriots eagerly pore over the readouts as they come in, then ask if you could land again to compare their scans to ones taken on the ground.`
+
+	on complete
+		payment 50000
+		conversation
+			`You breath a sigh of relief once you are back on the ground without a reaction from the Quarg. You were a touch worried they might take exception to being scanned, but they seem to consider it par for the course for this system filled with human engineers.`
+			`	Dawn does some follow-up scans of idle Quarg ships, and they take some time to make sure all their pictures are well focused and accurate. They let you know that they will meet you back in the spaceport bar.`
+
+mission "Remnant: Expanded Horizons Quarg 3"
+	name "Expanded Horizons Return"
+	description "Take the Remnant researchers to <planet>"
+	passengers 3
+	blocked `This mission requires bunks for three passengers.`
+	source "Wayfarer"
+	destination
+		government "Remnant"
+	to offer
+		has "Remnant: Expanded Horizons Quarg 2: done"
+	on offer
+		conversation
+			`Back in the spaceport bar you wait around for a couple hours. Just when you were about to go find them, the trio walk in looking a tad tired but otherwise unreadable.`
+			`	"OK, I think we are ready to go. Can you take us to <planet>?"`
+			choice
+				`	Yes`
+				`	Not yet`
+					defer
+			`	"Great. We will be waiting on your ship."`
+				accept
+	on complete
+		payment 50000
+		conversation
+			`Once you are back in the privacy of <ship>, the stony reserve fades to cheerfulness as the trio are a blur of motion as they chatter back and forth with their signs. Apparently this will be one of the biggest boosts to their research since the first time they salvaged the gear off a Korath raider.`
+			`	They eventually slow down their signs a bit and explain that up until the time the Remnant left human space, the Quarg had been reluctant to let anyone conduct detailed scans of their ships, let alone the close contact the engineers on Tarazed seem to enjoy. One of Dawn's fellow researchers continues with the explanation.`
+			`	"Our historical records just have a few pictures and videos of Quarg ships, along with a basic scan that doesn't provide more than the general outline of the ship. These scans," he says while tapping the databanks, "actually picked up detailed structural layouts and even identified a number of key components. It isn't nearly enough information to be able to duplicate them, but it is enough that we can start trying to figure out how they work.`
+			`	Upon arriving at <planet> they quickly thank you, and run back to their offices to finish prepping their material to publish and distribute.`
+
+mission "Remnant: Follow-up on Terminus Researchers"
```

Comment:
I'd take these missions out for now but keep them saved somewhere as to avoid conflicts with #3120. Once both that and this PR are in, we can look at addressing how the Remnant should know about the Deep knowing about them.

Reply:
OK. The Terminus Researcher missions have been removed, along with the "Busy Stranger" mission.